addons/popochiu/editor/popups/create_character/create_character.gd

```diff
@@ -49,7 +49,7 @@ func _create() -> void:
 	# ▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓
 	# Create the state Resource for the character and a script so devs
 	# can add extra properties to that state
-	var state_template: Script = load(CHARACTER_STATE_TEMPLATE)
+	var state_template: Script = load(CHARACTER_STATE_TEMPLATE).duplicate()
```

Comment:
I see what happens here, but to learn something solid... I guess you're working around the bug where you can create only one character?

Reply:
Yes. The problem was that on the process of creating our game objects the original template file was being overwritten (something that was not a problem in Godot 3.x). I forgot to put a comment to clarify what the change is intended for.